lib/pages/chat/chat.dart

```diff
@@ -1469,8 +1497,21 @@ class ChatController extends State<Chat>
     }
   }
 
-  void downloadFileAction(BuildContext context, Event event) =>
-      event.saveFile(context);
+  Future<String?> downloadFileAction(BuildContext context, Event event) async =>
+      await event.saveFile(context);
```

Comment:
This change is not need, right ? remove it

Reply:
[same answer ](https://github.com/linagora/twake-on-matrix/pull/695#discussion_r1340852372) duplicate